aws/resource_aws_ses_template_test.go

```diff
@@ -0,0 +1,193 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ses"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSSesTemplate_Basic(t *testing.T) {
+	name := acctest.RandString(5)
+	var template ses.Template
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckSesTemplateDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckAwsSesTemplateResourceConfigBasic1(name),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckSesTemplate("aws_ses_template.test", &template),
+					resource.TestCheckResourceAttr("aws_ses_template.test", "name", name),
+					resource.TestCheckResourceAttr("aws_ses_template.test", "html", "html"),
+					resource.TestCheckResourceAttr("aws_ses_template.test", "subject", "subject"),
+					resource.TestCheckResourceAttr("aws_ses_template.test", "text", ""),
+				),
+			},
+		},
+	})
+}
+
+func TestAccAWSSesTemplate_Update(t *testing.T) {
+	t.Skipf("Skip due to SES.UpdateTemplate eventual consistency issues")
```

Comment:
Can you please explain the effect of eventual consistency? We have many tests which can deal with this sort of situation, usually by retrying operations on a particular error code.

Reply:
I see. We don't have any good helpers for this _yet_, but I observed very similar behaviour in a few other existing tests, so we should probably create one at some point.